### PR TITLE
CRYP-162: bitcoin transaction details page

### PR DESCRIPTION
**Pull request by narroarrow**
Added BTC styling to the wallet overview page and transactions details page

image
image

### CODE REVIEW

File: packages/client/components/TransactionDetails.tsx [hunk 1]
```diff
@@ -127,31 +134,39 @@ export function TransactionDetails({ transaction, walletAddress }: Props) {
                         </Text>
                     </HStack>
                 </Box>
-                <Box style={styles.subItemWrapper}>
+                <Box style={currencyTypeCheck ? { display: "none" } : styles.subItemWrapper}>
```
Comment (jason-gerard) [inline packages/client/components/TransactionDetails.tsx:137]: Why are all of these being set to display none if its bitcoin?

**Review by domsec — APPROVED** [review]
Looks good. Fortunately we're only supporting 2 crypto's because this file is starting to become a bit messy. We could definitely refactor this in future sprints to incorporate inheritance/composition/specialization/components to make it more modular.

**Review by jason-gerard — APPROVED** [review]

**Comment by jason-gerard** [comment]
> Looks good. Fortunately we're only supporting 2 crypto's because this file is starting to become a bit messy. We could definitely refactor this in future sprints to incorporate inheritance/composition/specialization/components to make it more modular.

I already setup a function that returns the all the data based on the type of currency it is so we don't need all these conditionals, so we can probably just use that at some point.

File: packages/client/screens/WalletOverviewScreen.tsx [hunk 2]
```diff
@@ -42,7 +42,10 @@ export default function WalletOverviewScreen({ route, navigation }: Props) {
     }, []);
     return (
         <View style={styles.view}>
-            <Box style={styles.walletDetailsWrapper}>
+            <Box
+                style={styles.walletDetailsWrapper}
+                backgroundColor={currencyType == "BITCOIN" ? "rgba(247, 147, 26, 0.25)" : "rgba(60, 60, 61, 0.25)"}
```
[the remark this answers is not in the document]
Comment (narroarrow) [reply packages/client/screens/WalletOverviewScreen.tsx:47]: Using HEX and opacity separately here would apply the opacity to all contents so I think using the rgba function is best here to directly apply opacity to the backgroundColor

**Comment by narroarrow** [comment]
> 

Wasn't aware of that service, will implement it in the future.

**Pull request merged**